### DISCUSSION TITLE
LDAP Authentication: No admin credentials

### DISCUSSION THREAD

**Discussion by tobbeyy** [requarks/wiki#7467] [Help / Questions] [answered]
Hi,
I have a question concerning LDAP Authentication.

As we are part of a bigger organization it is impossible for us to get the admin credentials for the LDAP server. But we can still use anonymous binds and check if a certain account exists and is part of a group etc. However, the fields for mail and name are empty when doing an anonymous request (which makes sense).
You can still get the detailed profile information from the LDAP server if you use the login credentials of the account as the admin credentials. Then you have access to all the information about this account but obviously not about other accounts.
(I'm sorry if this explanation is clumsy, I have no experience in LDAP)

Assuming we cannot change this LDAP setup: Can we still use LDAP as a strategy in wiki.js?
Is there maybe a way to manually add user information (mail, name) and only do authentication over LDAP?
Or is wiki.js capable of using varying admin credentials (always using the credentials from the current login process) and extracting the user information this way?
Or it there any other workaround?

Thank you in advance :)

```
OS: Debian 12 and Docker
Wiki.js version: 2.5.305
Database engine: PostgreSQL 15.10
```

**Comment by NGPixel** [comment, marked as answer]
No, email is a hard requirement for all external strategies.
